docs/admin-guide/server.md

```diff
@@ -29,6 +31,15 @@ MAIL_USE_SSL = False
 # If null no badge is displayed in scout
 ACCREDITATION_BADGE = "swedac-1926-iso17025.png"
 
+# LDAP login Settings
```

Comment:
There were a bunch on missing params here, so I've just copy-pasted the current confg file